src/test/parse-fail/mod_file_not_exist.rs

```diff
@@ -8,6 +8,8 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
+// ignore-windows
```

Comment:
@BurntSushi I considered different approaches - this could maybe be turned into a run-make test instead.

I went for this solution, because I looked at `src/test/codegen/dllimports/main.rs` (and others) that do windows-specific things by exhausting the other options.